js/script.js

```diff
@@ -520,55 +521,40 @@ const checkQuestion = () => {
 }
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
-// Show the correct alert message for different categories
+// Show the correctSwal.fire message for different categories
 const { attribute, category, value } = currentQuestion
 
 if (category === 'accessories') {
   if (keep) {
     sound()
-    alert(
-      `Yes! Guess is correct, secrect person wears ${attribute}! Keep all who wears ${attribute}.`
-    )
+    Swal.fire(`Yes! Guess is correct, secrect person wears ${attribute}!`, `Keep all who wears ${attribute}.`)
   } else {
     sound()
-    alert(
-      `Wrong guess, the person doesn't wear ${attribute}! Remove all who wears ${attribute}.`
-    )
+    Swal.fire(`Wrong guess, the person doesn't wear ${attribute}`,`Remove all who wears ${attribute}.`)
   }
 } else if (category === 'hair') {
   if (keep) {
     sound()
-    alert(`Yes! Guess is correct, the person has ${value} hair! Keep all who has ${value} hair.`
-    )  
+    Swal.fire(`Yes, the person has ${value} hair!`) 
   } else {
     sound()
-    alert(
-    `Wrong guess, the person doesn't have ${value} hair. Remove all who has ${value} hair.`
-    )
+    Swal.fire(`No, the person hasn't ${value} hair!`)
   }
 } else if (category === 'eye') {
   if (keep) {
     sound()
-    alert(
-      `Yes! Guess is correct, the person has ${value} eyes! Keep all who has ${value} eyes.`
-    )
+    Swal.fire(`Yes! Guess is correct, the person has ${value} eyes!`, `Keep all who has ${value} eyes.`)
   } else {
     sound()
-    alert(
-      `Wrong guess, the person doesn't have ${value} eyes. Remove all that has ${value} eyes.`
-      )
+   Swal.fire(`Wrong guess, the person doesn't have ${value} eyes.`, `Remove all that has ${value} eyes.`)
   }
 } else if (category === 'other') {
   if (keep) {
     sound()
-    alert(
-      `Yes! Guess is correct, the person has a ${attribute}. Keep all who has a ${attribute}.`
-    )
+   Swal.fire(`Yes! Guess is correct, the person has a ${attribute}.`, `Keep all who has a ${attribute}.`)
 } else {
   sound()
-    alert(
-      `Wrong guess, the person doesn't have a ${attribute}. Remove all that has a ${attribute}.`
-    )
+   Swal.fire(`Wrong guess, the person doesn't have a ${attribute}.`, `Remove all that has a ${attribute}.`)
   } 
 }
   // filter to keep or remove based on the keep variable.
```

Comment:
Good job with the filterfunction, interesting to see that it can be written like that